### ISSUE TITLE
[2] Theming fixes for upgrade to hyku 3.0

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#147]
# summary
Several styles are not matching up between dev & prod

1. the search bar isn't styled correctly.
2. facets are different colors
3. fonts are different
4. sizing etc of header is different

# acceptance criteria
- [ ] dev uses all the same styles that are on prod

# demo
Prod (search bar): 

![Image](https://user-images.githubusercontent.com/1054448/205061090-0f8e0b50-2a6d-4a02-8ddc-eabdf049db59.png)

Dev: 

![Image](https://user-images.githubusercontent.com/1054448/205061122-1e610bf5-7001-4434-bcdf-58640e750803.png)

**Comment by orangewolf** [comment]
Linked Items
---
[EPIC 1 - BULKRAX && TESTING](https://gitlab.com/notch8/louisville-hyku/-/issues/25)

**Comment by orangewolf** [comment]
Summer February 2022
---
I just noticed that some of the styles that apply to their search bar, facets, etc (which seem to be missing from dev/staging) are coming from what looks like an inline stylesheet in the inspector. I tried to see if an inline style had been overridden in that file, and it had not. the only other possible thing I can think of would be that they have some rules in their custom CSS settings which are applying on prod but not on staging/dev
We will need to get in to their dashboard on prod to copy those over, so since we don’t have an admin login for prod at this point maybe we can ask them for those credentials in the next meeting. adding this to the ticket as well


![Image](https://user-images.githubusercontent.com/1054448/205061323-13615b82-7655-45c9-b1ae-025c4b7ee24b.png)
